### DISCUSSION TITLE
A4A_Economic_Improvement_Proposal_2

### DISCUSSION THREAD

**Discussion by mashvires** [Tradecoin-Token/A4ADAO#15] [Ideas]
For more information on this proposal, please go to: https://medium.com/@A4A.DAO/a4a-tokenomics-upgrade-60fd263abb89

**PROPOSAL**

A4ADAO proposes to change A4A BRONZE tier in 5 phases. It believes its function as lower starting barrier for CODE Expansion has deprecated. Each phase will have an approximate cycle of 90 days after the initial phase.

1st Phase in December 21, 2022 – Blockheight: 2470000
-BRONZE 100 A4A – 999.999 A4A

2nd Phase 1st April, 2023
-BRONZE 200 A4A – 999.999 A4A

3rd Phase 1st July, 2023
-BRONZE 300 A4A – 999.999 A4A

4rd Phase 1st October, 2023
-BRONZE 400 A4A – 999.999 A4A

5th Phase 1st December, 2023
-BRONZE 500 A4A – 999.999 A4A

Initial voting will begin Wednesday, December 7, 2022 - UTC 12:00.
Voting will last 14 days until Wednesday, December 21, 2022 - UTC 12:00.

**If approved. Proposed tiers will be implemented on Thursday December 22, 2022 - Blockheight: 2470000.**

A4A_Economic_Improvement_Proposal_2

Required Quorum: 51% A4A
How to vote:
Send a transaction of 0.00001 A4A with the attachment format:
"EIP1 Y"
or
"EIP1 N"
to:
3JhE2u6TTeZ9uim3799JxnZ9e9tMUfiCMCA

**Proposed changes scheduled to take effect at blockheigh: 2470000**

**Comment by Doge-Miguel** [comment]
So far this sounds like a decent idea, lending a hand to those who are beginning to build a CODE stack so that they may not be dumped upon if one wallet decides to play dirty. 20usdt will help the DAO immensely just from one person and multiple user seeing a bright future can make a large impact on the treasury holdings. As long as treasury amounts are fully transparent and accurate. I look forward to reading others opinions and suggestions. I am always learning and grateful to be here
